### ISSUE TITLE
❓ [Immich] Base version is out of date

### ISSUE THREAD

**Issue by celynw** [alexbelgium/hassio-addons#892]
I noticed the Immich addon is a few versions behind the main repository now, `1.57` versus `1.66`.
The reason this is a problem, is because the android app auto-updates, and the version of the server ought to be close to the version of the client app. Development is moving fast over there!

I've been reading the issues and maybe there's a good reason for this, but I'm not really sure. Actually, I'm not sure how any of the addons in this repository get updated, surely it can't be a manual process? Is there something I can help with?

According to the README on the immich addon page, it's based on the [docker image from imagegenius](https://github.com/imagegenius/docker-immich), which is up-to-date with upstream.

**Comment by alexbelgium** [comment]
If I remember correctly the upstream dev removed embedded postgresql internal support which globally means that everyone will lose their database and need to switch to the external postgres add-on I've made (but not tested)

I had paused the updates to postpone those issues...

**Comment by alexbelgium** [comment]
I've restarted automatic updates! Beware that using the built-in Postgres 14 will likely fail at some point. It is recommended to upgrade to an external Postgres 15 database. A solution is the specific addon I've built.

**Comment by ElVit** [comment]
I can confirm that the immich addon is working with postgres addon created by you @alexbelgium.
Thanks for that 👍🏼 

In the beginning I had some troubles to get it running because I had set some options in the postgres addon wrong.
After I corrected these options it seems that (some or all?) of my changes had no effect.
Only after I reinstalled the postgres addon and started it with the correct options, the immich addon could access the postgres database.

I think maybe the right way would be to change the settings in the Postgres UI. 
But unfortunatelly I could not access the Postgres UI.

**Comment by alexbelgium** [comment]
Wow great thanks!!! I had just done some minor tests with it but nothing serious. Glad it works. What were the options that you had to correct ? So that I can see if I can do something (even it the "something" is just to reset the database if the options changed)

thanks!

**Comment by ElVit** [comment]
I think it was mainly the username that was wrong. But probably also the database name, I'm not really sure.
For the password I used a secret variable, so this shouldn't be the issue.
Oh and I also tried to delete the postgres conf file, but unfortunately it didn't helped.

BTW: Do you know a way to migrate the old (internal) immich database to the new (external) database?
Or maybe there is a way to simply reimport the existing photos?
